.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,15 +6,28 @@ labels: 'needs: discussion'
 assignees: ''
 
 ---
+<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->
 
-Those who maintain this project do so as volunteers and we have no control over their schedule or priorities.
+### Proposed change
+<!-- Use this section to describe the feature you'd like to be added. -->
 
-This means if the feature you are proposing is something you'd like to see added the best way to achieve that is for you to organise the effort required to build it. This could be evangelising for the feature, paying someone, doing it yourself, etc.
 
-Even so it might never get merged because the trade off between additional happy users and maintenance burden is not favourable.
+### Alternative options
+<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+### Who would use this feature?
+<!-- Describe the audience for this feature. This information will affect how work is assigned across the maintenance and contributing team. -->
+
+
+### How much time will adding it take?
+<!-- Try to estimate how much work adding this feature will require. This information will affect how work is assigned across the maintenance and contributing team. -->
+
+
+### Who can do this work?
+<!-- What skills are needed? Who can be recruited to add this feature? This information will affect how work is assigned across the maintenance and contributing team. -->
+
+
+### Thank you maintainers!
```

Comment:
Remove? 😭 

Give me an emoji vote on getting rid of this section :+1: to keep, :-1: to remove.

Reply:
I'm also leaning towards removing this, since this issue template is already a bit long and so it may not have the effect we want. I'd love to find another way to let people give positive feedback to the community - maybe an on-going thread in Discourse or something like this?

Reply:
Nods. I think the thanks has to be genuine and heartfelt. Otherwise it can be a negative thing as well. (Remember that guy who read that pop-sci article on how people like hearing their own name, that it is a good way to remember people's names and so ends up using your name every second sentence?)

This makes it hard to channel into anything "standard" because ... well if it is standard then it doesn't mean anything, it just becomes transactional (like saying thanks when I pass you the butter at dinner). So maybe a reminder or pointer to nudge people towards the right behaviour?